flexeval/core/evaluate_multiple_choice.py

```diff
@@ -104,6 +106,9 @@ def evaluate_multiple_choice(
     metrics_dict: dict[str, float] = {
         "accuracy": accuracy,
         "byte_norm_accuracy": byte_norm_accuracy,
+        "f1_score": f1_score(
+            [res["prediction"] for res in results], [res["answer_index"] for res in results], average="macro"
+        ),
```

Comment:
```suggestion
        "macro_f1_score": f1_score(
            [res["prediction"] for res in results], [res["answer_index"] for res in results], average="macro"
        ),
```
として、マクロF1であることを明示するのはどうでしょうか。
合わせて、micro F1も計算しておいてログに含めませんか？